src/pages/programIndicators/form/AnalyticsPeriodBoundaries.module.css

```
@@ -0,0 +1,26 @@
+.analyticsPeriodBoundaryForm {
+    display: flex;
+    flex-direction: column;
+    gap: var(--spacers-dp8);
+}
+
+.analyticsPeriodBoundaryContainer {
+    border: 1px solid var(--colors-grey400);
+    border-radius: 4px;
+    display: flex;
+    margin-block: 8px;
```

Minor, but variables could we used here and in other places
```suggestion
    margin-block: var(--spacers-dp8);
```